BUILDING.md

```diff
@@ -23,7 +23,6 @@ These are the essentials tools to build RPCS3 on Linux. Some of them can be inst
 * [CMake 3.16.9+](https://www.cmake.org/download/)
 * [Qt 5.15.2](https://www.qt.io/download-qt-installer)
 * [Vulkan SDK 1.2.154+](https://vulkan.lunarg.com/sdk/home) (See "Install the SDK" [here](https://vulkan.lunarg.com/doc/sdk/latest/linux/getting_started.html))
-* [SDL2](https://www.libsdl.org/download-2.0.php) (for the FAudio backend)
```

Comment:
If you really remove all those backends then there's a lot more mentions of these libs in this file

Reply:
Cubeb uses alsa and pulse internally, so i dont't think removing them is a good idea